frontend/src/routes/FacilityBooking/ViewBooking/index.tsx

```diff
@@ -145,17 +146,17 @@ export default function ViewBooking() {
           if (data.data === '' || data.data === undefined) {
             console.log(data.err)
           } else {
-            openTelegram(data.data)
+            openTelegram(data.data, newTab)
           }
         })
     } catch (err) {
       console.log(err)
     }
   }
 
-  const openTelegram = (userID) => {
+  const openTelegram = (userID, tab) => {
     const site = 'https://telegram.me/' + userID
-    window.open(site)
+    if (!!tab) tab.location.href = site
```

Comment:
why use `!!` ?

Reply:
tab is nullable, so need check it isnt. !! converts to boolean